tfsdk/attribute_plan_modification.go

```diff
@@ -118,6 +119,85 @@ func (r RequiresReplaceIfModifier) MarkdownDescription(ctx context.Context) stri
 	return r.markdownDescription
 }
 
+// PreserveState returns a PreserveStateModifier.
+func PreserveState() AttributePlanModifier {
```

Comment:
Naming nit (I know the worst kind! 😦 ): seeing this in a [provider implementation](https://github.com/hashicorp/terraform-provider-corner/pull/32) and with my new provider developer hat on, I initially was left wondering "What are the other attributes doing then, if they aren't preserving state? Is data being deleted? Is this scary?" As a framework developer, I happen to know this is described by the [Terraform Resource Instance Change Lifecycle](https://github.com/hashicorp/terraform/blob/main/docs/resource-instance-change-lifecycle.md), but I'm wondering if there is anything minor we could or should do to further drive home when it might be used. Maybe `PreserveComputedState()` is what I'm reaching for naming-wise?

Reply:
How do you feel about `tfsdk.UseStateForUnknown`?